openmls/src/group/public_group/diff/compute_path.rs

```diff
@@ -76,7 +77,7 @@ impl<'a> PublicGroupDiff<'a> {
                             Ciphersuite::MLS_256_DHKEMP384_AES256GCM_SHA384_P384,
                             Ciphersuite::MLS_128_X25519KYBER768DRAFT00_AES128GCM_SHA256_Ed25519,
                         ]),
-                        Some(&[ExtensionType::PerDomainTrustAnchor]),
```

Comment:
Why not keeping PDTA as an extension lying around but simply remove it from RequiredCapabilities? And then on the CC front we remove them.

Reply:
Because I was pretty confident we will never have to deal with it in the future. Also, it adds some maintenance costs. That being said it's useful for testing purposes as it's the only non-standard extension we have currently.
I'll do what you suggest